src/Handlers/CardsPaymentMethodHandler.php

```diff
@@ -401,7 +414,7 @@ public function preparePaymentsRequest(
             ),
             $transaction->getOrder()->getOrderNumber(),
             $this->configurationService->getMerchantAccount(),
-            $transaction->getReturnUrl(),
+            'test url',
```

Comment:
Expected?

Reply:
Thanks for noticing, I will put back the original one and if it's necessary we can adjust it later to a new url